app/services/letter.py

```
@@ -0,0 +1,70 @@
+"""Generate appeal letters based on case information."""
+from __future__ import annotations
+
+from datetime import date
+from typing import Iterable, Optional
+
+from jinja2 import Environment, Template
+
+DEFAULT_TEMPLATE = Environment(autoescape=False).from_string(
```

Ruff (0.13.1)

Using jinja2 templates with `autoescape=False` is dangerous and can lead to XSS. Ensure `autoescape=True` or use the `select_autoescape` function.

(S701)